archivers/wimlib/Portfile

```diff
@@ -29,21 +29,32 @@ master_sites        ${homepage}/downloads/
 depends_build       port:pkgconfig
 
 depends_lib         port:libxml2 \
-                    port:ntfs-3g \
-                    path:lib/libssl.dylib:openssl \
-                    port:osxfuse
+                    path:lib/libssl.dylib:openssl
 
 depends_run         port:cabextract \
                     port:cdrtools \
                     port:mtools
 
-# ntfs-3g is not universal
-universal_variant   no
-
 configure.args      --disable-silent-rules \
+                    --with-libcrypto \
                     --without-fuse \
-                    --with-ntgs-3g \
-                    --with-libcrypto
+                    --without-ntfs-3g
+
+variant fuse description {Mount WIM images with FUSE} {
+  depends_lib-append      port:osxfuse
```

Comment:
The indentation of the lines in the variants you added does not conform to the modeline at the top of the portfile. Indentations should be at multiples of 4 spaces.